File: articles/algorithms-before-symbols-verbal-procedures-in-medieval-mathematics/c/verbal_sequence.c
#include <stdio.h>

int main(void) {
    const char *steps[] = {"take the number", "double it", "add the adjustment", "check the result"};
    for (int i = 0; i < 4; i++) {
        printf("%d: %s\n", i + 1, steps[i]);
    }
    return 0;
}

File: articles/algorithms-before-symbols-verbal-procedures-in-medieval-mathematics/java/VerbalSequence.java
public class VerbalSequence {
    public static void main(String[] args) {
        String[] steps = {"take the number", "double it", "add the adjustment", "check the result"};
        for (int i = 0; i < steps.length; i++) {
            System.out.printf("%d: %s%n", i + 1, steps[i]);
        }
    }
}
